tests/title-search-test.js

```diff
@@ -9,16 +9,42 @@ describeApplication('TitleSearch', () => {
   let titles;
 
   beforeEach(function () {
-    titles = this.server.createList('title', 3, 'withPackages', {
+    titles = this.server.createList('title', 3, 'withPackages', 'withSubjects', 'withIdentifiers', {
       name: i => `Title${i + 1}`,
-      publicationType: i => (i % 2 ? 'book' : 'journal')
+      publicationType: i => (i % 2 ? 'book' : 'journal'),
```

Comment:
We use this `i % 2` statement multiple times now, we should add a comment explaining that only odd indexed items are given these alternate attributes.